cmake/O2DefineOptions.cmake

```diff
@@ -28,5 +28,5 @@ function(o2_define_options)
   # for the complete picture of how BUILD_SIMULATION is handled see
   # ../dependencies/O2SimulationDependencies.cmake
 
-  option(ENABLE_UPGRADES "Enable detectors for upgrades" OFF)
```

Comment:
should not be changed. You'll be able to set this from the outside

Reply:
Option has been removed

Reply:
sorry. There was a misunderstanding. The declaration of this option is still necessary here (in my understanding). Otherwise you can't modify from the outside. What I wanted to say is maybe that this file should not be touched since the option is available correctly on the dev branch already.

Reply:
ok, Reverted